webhook_server/tests/test_runner_handler.py

```
@@ -1889,6 +1910,398 @@ async def run_command_side_effect(command: str, **kwargs: Any) -> tuple[bool, st
+        return mock_pr
+
+    @pytest.mark.asyncio
+    async def test_amend_when_email_mismatch(self, runner_handler: RunnerHandler) -> None:
+        """Original PR has Signed-off-by, cherry-pick author differs — amend."""
+        mock_pr = self._make_pr_with_commits(
+            ["[Storage] Update owners\n\nSigned-off-by: Jenia Peimer <jpeimer@redhat.com>\n"],
+            author_name="Jenia Peimer",
+            author_email="jpeimer@redhat.com",
+        )
+
+        async def run_command_side_effect(command: str, **kwargs: Any) -> tuple[bool, str, str]:
```

Ruff (0.15.10)

[warning] Unused function argument: `kwargs`

(ARG001)